ansible_collections/arista/avd/docs/installation/collection-installation.md

```diff
@@ -1,14 +1,19 @@
-# Collection Installation
+# Collection installation
 
-These instructions are for regular users to install via Ansible Galaxy. To setup a development environment use [these](../contribution/setup-environment.md) instructions.
+## Installation workflow
 
-**arista.avd** can also be consumed using the ["AVD All-in-one" container](https://github.com/arista-netdevops-community/avd-all-in-one-container).
+- Python **3.8** or later
+- ansible-core from **2.11.3** to **2.12.x** excluding **2.12.0** to **2.12.5**
```

Comment:
Should we link to an Ansible installation guide?

Reply:
yes, good idea!